src/main/java/com/bobocode/restApiDemo/entity/Camera.java

```diff
@@ -0,0 +1,13 @@
+package com.bobocode.restApiDemo.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+public class Camera {
+  @JsonProperty("id")
+  private Integer nasaId;
+
+  @JsonProperty("name")
```

Comment:
If json property named the same as class field name, than `@JsonProperty` is not required